{{ cookiecutter.__project_name_kebab }}/pyproject.toml

```diff
@@ -19,23 +19,19 @@ classifiers = [
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
     "Framework :: Django",
-    "Framework :: Django :: 3.0",
-    "Framework :: Django :: 3.1",
     "Framework :: Django :: 3.2",
-    "Framework :: Django :: 4.0",
-    "Framework :: Django :: 4.1",
     "Framework :: Django :: 4.2",
     "Framework :: Wagtail",
-    "Framework :: Wagtail :: 2",
     "Framework :: Wagtail :: 4",
     "Framework :: Wagtail :: 5",
 ]
 requires-python = ">=3.8"
 dynamic = ["version"]
 dependencies = [
     "Django>=3.2",
-    "Wagtail>=4.1,<6.0"
+    "Wagtail>=4.1"
```

Comment:
I converted this to an open range because now that the version of wagtail on `main` is `6.0a0`, if you try to run the `wagtailmain` toxenv, pip throws

```
ERROR: Cannot install wagtail 6.0a0 (from git+https://github.com/wagtail/wagtail.git) and generated-package-name[testing]==0.1.0 because these package versions have conflicting dependencies.
```

trying to install. Open to other ideas, but I think an open range is reasonable here.

Reply:
open range is what I started advocating for